grails-app/services/com/coinCollector/CustomerService.groovy

```diff
@@ -11,84 +11,105 @@ import utils.phoneNumber.PhoneNumberUtils
 
 @Transactional
 class CustomerService {
+    public Map parsedParams(Map params) {
+
```

Comment:
É bom vc já se acostumar não manter linhas em branco no inicio e fim dos metodos, pode remover?

Reply:
Removido linhas em branco no início e fim dos métodos.